lib/super_digit.rb

```diff
@@ -1,15 +1,29 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity O(n) because I am using a loop ?
+# Space Complexity 0(n) becuase I am using recursion?
 def super_digit(n)
```

Comment:
This works, but the time/space complexity is either O(log10 n) or O(n) if n is the number of digits.